crates/apub/src/objects/community.rs

```diff
@@ -142,21 +141,16 @@ impl Object for ApubCommunity {
 
     // Fetching mods and outbox is not necessary for Lemmy to work, so ignore errors. Besides,
     // we need to ignore these errors so that tests can work entirely offline.
-    let fetch_outbox = group.outbox.dereference(&community, context);
-    let fetch_followers = group.followers.dereference(&community, context);
-
-    if let Some(moderators) = group.attributed_to {
-      let fetch_moderators = moderators.dereference(&community, context);
-      // Fetch mods, outbox and followers in parallel
-      let res = tokio::join!(fetch_outbox, fetch_moderators, fetch_followers);
-      res.0.map_err(|e| debug!("{}", e)).ok();
-      res.1.map_err(|e| debug!("{}", e)).ok();
-      res.2.map_err(|e| debug!("{}", e)).ok();
-    } else {
-      let res = tokio::join!(fetch_outbox, fetch_followers);
-      res.0.map_err(|e| debug!("{}", e)).ok();
-      res.1.map_err(|e| debug!("{}", e)).ok();
-    }
+    let community_ = community.clone();
+    let context_ = context.reset_request_count();
+    spawn_try_task(async move {
+      group.outbox.dereference(&community_, &context_).await?;
```

Comment:
You don't want to ignore the errors anymore?

Reply:
The spawn_try_task has its own error handling and logs them, so there is no need to ignore them explicitly.